tests/automation.js

```diff
@@ -63,6 +63,7 @@ var gfxTests = [
   { name: "gfx/CreateImmutableCopyTest", maxDifferent: 0 },
   { name: "gfx/TestLauncher", maxDifferent: 0 },
   { name: "gfx/MediaImageTest", maxDifferent: 0 },
+  { name: "gfx/TextEditorGfxTest", maxDifferent: 592 },
```

Comment:
(And the text wraps in a different spot, which might just be the result of a different font size.)